docs/swagger.go

```diff
@@ -1,14 +1,15 @@
 package docs
 
 import (
+	"embed"
 	"strings"
 
 	"github.com/gin-gonic/gin"
-	"github.com/gobuffalo/packr/v2"
 	"github.com/gotify/location"
 )
 
-var box = packr.New("docs", "./")
+//go:embed spec.json
+var box embed.FS
```

Comment:
It's possible to directly inject a single file into a string, then we don't have the overhead with the embed.FS, the import of embed is still needed.
```go
import (
	_ "embed"
	"strings"

	"github.com/gin-gonic/gin"
	"github.com/gotify/location"
)

//go:embed spec.json
var spec string
```